javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,9 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  const wordsWithSuffix = [];
+  for (let i = 0; i < words.length; i++) {
+    words[i] += suffix;
+    wordsWithSuffix.push(words[i]);
```

Comment:
I would combine these two lines. Right now you're manipulating `words[i]` directly, but I would just use this as a reference rather than manipulating it directly. This way you're not adjusting the array value directly, but using it to create a new value.

`wordsWithSuffix.push(words[i] + suffix)`